src/listener/handlers/document-update-handler.ts

```diff
@@ -155,10 +156,13 @@ class DocumentUpdateHandler implements IHandler {
                     const fileName = doc['fileName'];
                     const fileFormat = doc['content']['file_core']['format'];
 
-                    const fileExt = fileName.substring(fileName.indexOf('.') + 1);
+                    const acceptedFileFormatForValidation = Object.keys(config.get("FILE_VALIDATION_IMAGES"));
 
-                    if (fileFormat != fileExt && fileFormat && fileExt) {
-                        const msg = `The file extension, ${fileExt}, of the file with filename, "${fileName}", doesn't match the file format, "${fileFormat}", in the metadata.`;
+                    if ((fileFormat && !fileFormat.match(/^[a-zA-Z]/))
```

Comment:
I would split the predicate to 3 different predicates on the "or" part for the sake of readability.

Reply:
I refactored it.